src/context/cart/Context.jsx

```diff
@@ -71,10 +68,11 @@ const CartProvider = ({ children }) => {
 
   const removeFromCart = async (product) => {
     try {
-      const res = await axios.delete(
-        `/api/user/cart/${product._id}`,
-        headerConfig
-      );
+      const res = await axios.delete(`/api/user/cart/${product._id}`, {
+        headers: {
+          authorization: JSON.parse(localStorage.getItem("user-token")),
+        },
+      });
```

Comment:
just a suggestion for headers object, can make separate variable, so can directly use it